src/components/manager/runs/RunLastContent.js

```diff
@@ -113,13 +124,13 @@ const processIngestions = (data, currentTables) => {
                 : [fileNameFromPath(filePaths)];
 
             const dateStr = getDateFromEndTime(run.details.run_log.end_time);
-            const date = Date.parse(dateStr);
+            const date = Date.parse(run.details.run_log.end_time);
 
-            const currentIngestion = { date: dateStr, dataType, tableId, fileNames };
+            const currentIngestion = { date: dateStr, dataType, tableId, fileNames, newdate: date };
```

Comment:
`newdate` is not a very informative property name I think - what does it do differently vs dateStr?